sklearn/discriminant_analysis.py

```diff
@@ -599,7 +616,7 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
     >>> clf.fit(X, y)
     ... # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
     QuadraticDiscriminantAnalysis(priors=None, reg_param=0.0,
-                                  store_covariances=False, tol=0.0001)
+                   store_covariance=False, store_covariances=None, tol=0.0001)
```

Comment:
PEP8 visual indentation would have this line aligned with the opening parenthesis